fearless/Modules/Staking/SelectValidatorsFlow/YourValidatorList/YourValidatorListPresenter.swift

```diff
@@ -47,8 +50,8 @@ final class YourValidatorListPresenter {
         }
 
         do {
-            let sections = try viewModelFactory.createViewModel(for: model)
-            view?.reload(state: .validatorList(sections))
+            let viewModel = try viewModelFactory.createViewModel(for: model, locale: selectedLocale)
+            view?.reload(state: .validatorList(viewModel: viewModel))
         } catch {
             logger?.error("Did receive error: \(error)")
```

Comment:
and the view remains empty. maybe `view?.reload(state: .error)`

Reply:
done